src/styles/block/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  padding: 0 50px;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  height: 60px;
+  margin-left: 20px;
+  align-items: center;
+  color: $primary-color;
```

Comment:
Variables for colors should be used consistently. It seems `$primary-color` is not defined in this file. Please ensure that all color variables are defined in `variables.scss` and imported into this file to maintain consistency and readability.